src/discussions/discussions-home/InformationsBanner.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+
+import { useSelector } from 'react-redux';
+
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { Hyperlink, PageBanner } from '@edx/paragon';
+
+import { selectUserIsStaff, selectUserRoles } from '../data/selectors';
+import messages from '../messages';
+
+function InformationBanner({
+  intl,
+}) {
+  const [showBanner, setShowBanner] = useState(true);
+  const userRoles = useSelector(selectUserRoles);
+  const isAdmin = useSelector(selectUserIsStaff);
+  const learnMoreLink = 'https://openedx.atlassian.net/wiki/spaces/COMM/pages/3509551260/Overview+New+discussions+experience';
+  const TAFeedbackLink = process.env.TA_FEEDBACK_FORM || 'https://test-ta-feedback';
+  const staffFeedbackLink = process.env.STAFF_FEEDBACK_FORM || 'https://test-staff-feedback';
```

Comment:
We should keep the feedback link empty string Instead of redirecting to a dummy feedback link. `process.env.TA_FEEDBACK_FORM || ''`. We can add a check if the `TAFeedbackLink || staffFeedbackLink` is available then display the link which will make more sense. Redirecting to a nonuseable link is not a good idea. 
`TAFeedbackLink || staffFeedbackLink && (
<Hyperlink
          destination={showStaffLink ? staffFeedbackLink : TAFeedbackLink}
          target="_blank"
          showLaunchIcon={false}
          variant="muted"
          isInline
        >
          {intl.formatMessage(messages.shareFeedback)}
        </Hyperlink>
        )`

Reply:
resolved env load issue